src/Select.js

```diff
@@ -998,6 +1012,24 @@ export default class Select extends Component<Props, State> {
       return <input name={name} type="hidden" value={value} />;
     }
   }
+  getCommonProps() {
```

Comment:
I think we should experiment with context here, especially after depending on React 16.3.